src/progressbar/progressbar.spec.ts

```diff
@@ -0,0 +1,116 @@
+import {iit, it, ddescribe, describe, expect, injectAsync, TestComponentBuilder} from 'angular2/testing';
+
+import {Component} from 'angular2/angular2';
+
+import {NgbProgressbar} from "./progressbar";
+
+function getBarWidth(nativeEl): number {
```

Comment:
That is something to research. `Element` doesn't have `style` on it so it fails. That being said, it returns a string, not a number.